ui/packages/catalog-ui-search/src/main/webapp/js/model/User.js

```diff
@@ -207,9 +207,14 @@ User.Preferences = Backbone.AssociatedModel.extend({
     this.listenTo(this, 'change:mapHome', this.savePreferences)
   },
   get(attr) {
-    const value = Backbone.AssociatedModel.prototype.get.call(this, attr)
-    if (attr === 'resultFilter' && value && value.type === 'NOT') {
-      return cql.simplify(value)
+    let value = Backbone.AssociatedModel.prototype.get.call(this, attr)
+    if (attr === 'resultFilter') {
+      if (typeof value === 'string') {
```

Comment:
So, if I am following correctly, the thought is, if we get here.... we have a resultFilter of type string, from a previous version... otherwise it should be an CQL object and we revert to the previous logic?

Reply:
Yes.